docker/Dockerfile

```diff
@@ -0,0 +1,22 @@
+ARG PG_MAJOR_VER=15
+
+FROM postgres:$PG_MAJOR_VER AS build
+
+ARG PG_MAJOR_VER
+
+RUN apt-get update \
+    && apt-get install build-essential postgresql-server-dev-$PG_MAJOR_VER -y
+
+ADD https://github.com/neondatabase/pg_embedding/archive/refs/heads/main.tar.gz main.tar.gz
```

Comment:
why not to copy from local disc? 

1. to run this docker file you need to clone the sources. I guess, that you’ll have it
2. It will allow you to build images on different branches/tags without changes